src/gui/search/searchwidget.cpp

```diff
@@ -129,7 +129,7 @@ void SearchWidget::fillCatCombobox()
     QList<QStrPair> tmpList;
     foreach (const QString &cat, m_searchEngine->supportedCategories())
         tmpList << qMakePair(SearchEngine::categoryFullName(cat), cat);
-    std::sort(tmpList.begin(), tmpList.end(), [](const QStrPair &l, const QStrPair &r) { return (l.first < r.first); } );
+    std::sort(tmpList.begin(), tmpList.end(), [](const QStrPair &l, const QStrPair &r) { return (QString::localeAwareCompare(l.first, r.first) < 0); } );
```

Comment:
I don't think QString::localeAwareCompare() will benefit us here. It is supposed to use the system locale for comparing, but what happens if the language chosen by the user(in qbt) is different than their locale? And I think locale benefits us when numbers or money amounts are involved(eg for decimal character). Here categories have only letters even when translated so I think operator `<` is fine here.